2d/CMakeLists.txt

```diff
@@ -27,19 +27,13 @@ if(build)
         "include/pcl/${SUBSYS_NAME}/impl/morphology.hpp"
         )
 
-    set(LIB_NAME "pcl_${SUBSYS_NAME}")
-    
     if(${VTK_FOUND})
         set(VTK_USE_FILE "${VTK_USE_FILE}" CACHE INTERNAL "VTK_USE_FILE")
         include("${VTK_USE_FILE}")
         set(VTK_IO_TARGET_LINK_LIBRARIES vtkCommon vtkWidgets vtkIO vtkImaging)
     endif(${VTK_FOUND})
 
     include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include" ${VTK_INCLUDE_DIRECTORIES})
-    PCL_ADD_LIBRARY("${LIB_NAME}" "${SUBSYS_NAME}" ${srcs} ${incs} ${impl_incs})
-    link_directories(${VTK_LINK_DIRECTORIES})
-    target_link_libraries("${LIB_NAME}" ${VTK_LIBRARIES} pcl_io)
-    PCL_MAKE_PKGCONFIG("${LIB_NAME}" "${SUBSYS_NAME}" "${SUBSYS_DESC}" "${SUBSYS_DEPS}" "" "" "" "")
```

Comment:
This should be `PCL_MAKE_PKGCONFIG_HEADER_ONLY`

Reply:
Can you send a new pull request? This is merged ;)

Reply:
OK